src/p5.dance.js

```diff
@@ -356,42 +356,49 @@ module.exports = class DanceParty {
 
 // Dance Moves
 
-  changeMoveLR(sprite, move, dir) {
-    if (!this.spriteExists_(sprite)) {
-      return;
-    }
+  getNewChangeMove(requested_change, current_move) {
```

Comment:
lets stick some @param tags on here

also isnt the style usually requestedChange and currentMove in our JS code?